File: go/wam/7606.go
const WamPinnedChatsMaxAlert = 7606 // channel: regular

type PinnedChatsMaxAlertEvent struct {
	AddToListSelected bool `wam:"add_to_list_selected"` // field 1
	PremiumStatus PREMIUMSTATUSTYPE `wam:"premium_status"` // field 2
	SubscribeSelected bool `wam:"subscribe_selected"` // field 3
}

File: ts/wam/7606.ts
export const WamPinnedChatsMaxAlert = 7606 as const

export interface PinnedChatsMaxAlertEvent {
  /** field 1, wire `add_to_list_selected` */
  addToListSelected?: boolean
  /** field 2, wire `premium_status` */
  premiumStatus?: PREMIUMSTATUSTYPE
  /** field 3, wire `subscribe_selected` */
  subscribeSelected?: boolean
}